docs/docs/getting-started/03-installation-and-launch.md

```diff
@@ -17,6 +17,16 @@ and add your user.
 
 :::
 
+:::caution
+
+Please note that Docker Desktop for Linux is currently **unsupported**, as its default configuration does not allow for 
+the elevated permissions required by the AICA System (e.g., for peripheral devices, graphics forwarding, ...). For full 
+compatibility, it is recommended to uninstall it. If you require Docker Desktop for other purposes, you may still use 
+the AICA System by following the provided instructions. However, additional steps may be needed when (manually) 
+interacting with AICA's Docker images and containers.
```

Comment:
How about this?

```suggestion
For Ubuntu users, please note that Docker Desktop for Linux is currently **unsupported**, as its default configuration
does not allow for the elevated permissions required by the AICA System (e.g., for peripheral devices, graphics
forwarding, ...). This interferes with some of the functionality of AICA Launcher, including attaching terminal or RViz
sessions to the AICA System container. For full compatibility, it is recommended to uninstall Docker Desktop entirely
and only install Docker Engine as described above.
```

Reply:
@eeberhard  oh sorry, I missed this comment when making the last commit's changes. The last commit is not proof-read, I wanted to verify  where we place the text. Let me know if you would like to keep some of the latest commit parts, but I think I'll try to include some of the text you changed here